1-Draft/RFCNNNN-Download-Cmdlet.md

```diff
@@ -0,0 +1,294 @@
+---
+RFC: RFC
+Author: Mark Kraus
+Status: Draft
+SupercededBy: 
+Version: 0.1
+Area: Utility
+Comments Due: 2018-06-01
+Plan to implement: Yes
+---
+
+# Download Cmdlet
+
+This RFC is to propose the design of a new cmdlet specialized for remote file downloads to compliment the existing Web Cmdlets.
```

Comment:
Just a spelling / homophone nit (the RFC is great): it should be _complement_, not _compliment_.